app/components/primer/label_component.rb

```diff
@@ -4,12 +4,12 @@ module Primer
   # Use labels to add contextual metadata to a design.
   class LabelComponent < Primer::Component
     NEW_SCHEME_MAPPINGS = {
-      primary: "Label--primary",
-      secondary: "Label--secondary",
-      info: "Label--info",
-      success: "Label--success",
-      warning: "Label--warning",
-      danger: "Label--danger"
+      primary: "Label--gray-darker",
+      secondary: "Label--gray",
+      info: "Label--blue",
+      success: "Label--green",
+      warning: "Label--yellow",
+      danger: "Label--red"
     }.freeze
 
     DEPRECATED_SCHEME_MAPPINGS = {
```

Comment:
I _believe_ we should instead be deleting this constant and leaving `NEW_SCHEME_MAPPINGS` in place, perhaps renaming it to `SCHEME_MAPPINGS`?

Reply:
I'm honestly confused at this point ;)

Deleting this mapping would be a breaking change. That's what we want?

Reply:
I think that's what we want in order to move this component to `Beta`. Perhaps we could start by raising some sort of warning in development/test if the deprecated mappings are used?

cc @manuelpuyol as I think he's worked on deprecating component options.

Reply:
`Primer::LabelComponent` has 179 uses in dotcom, maybe we should:
1. deprecate old options
2. open PRs in dotcom updating the uses
3. remove old options

Reply:
@manuelpuyol that sounds good. Might it be possible to use CI to do a runtime check to ensure we've removed all uses of the deprecated options?

Reply:
Assuming https://github.com/primer/view_components/pull/278 works, I think we want to close out this PR. We'd then work on removing the deprecated use cases in dotcom and open a follow up PR here removing the deprecated options. Sound right?

Reply:
yeah, we have to find a better deprecation method. I worked with this before https://github.com/primer/view_components/pull/40 but I'm not happy with how it turned out